docs/source/user-guide/general/drift_guide.rst

```diff
@@ -117,6 +117,22 @@ Cramer's V is always in the range [0,1], and is based on the `Pearson's chi-squa
 In general, it is recommended to use Cramer's V, unless your variable includes categories with a small number of samples (common practice is categories with less than 5 samples).
 However, in cases of a variable with many categories with few samples, it is still recommended to use Cramer's V, as PSI will not be able to detect change in the smaller categories.
 
+Detecting Drift in Unbalanced Classification Tasks
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In classification problems, it is common to have unbalanced data, meaning that the number of samples in each class is
+highly skewed. For example, in a dataset of credit card transactions, the number of fraudulent transactions is usually
+much lower than the number of non-fraudulent transactions, and can be below 1% of the total number of samples.
+
+In such cases, running the :doc:`TrainTestLabelDrift </checks_gallery/tabular/train_test_validation/plot_train_test_label_drift>`:
+or :doc:`TrainTestPredictionDrift </checks_gallery/tabular/train_test_validation/plot_train_test_prediction_drift>` checks
+with the default parameters will likely lead to a false negative, as for example a change in the percent of fraudulent
+transactions from 0.2% to 0.4% will not be detected, but may in fact be very significant for our business.
+
+To detect this kind of drift, set the ``balance_classes`` parameter of these checks to True. This will cause the check
+to consider all classes equally, regardless of their size.
+
+**Note**: You must also set the ``categorical_drift_method`` parameter to ``'cramers_v'`` in order to use this method.
```

Comment:
Why don't we set it explicitly then?

Reply:
as said in person, cramer's v is the default

Reply:
So phrase it like "make sure categorical_drift_method is set to the default value of cramers_v"

Reply:
Fixed